docs/sphinx/manual/writing_single_node_hpx_applications.rst

```diff
@@ -482,11 +482,11 @@ Parallel algorithms
      * Copies the elements from a range to a new location for which the given predicate is ``false``
      * ``<hpx/algorithm.hpp>``
      * :cppreference-algorithm:`remove_copy`
-   * * :cpp:func:`hpx::parallel::v1::replace`
+   * * :cpp:func:`hpx::replace`
      * Replaces all values satisfying specific criteria with another value.
      * ``<hpx/algorithm.hpp>``
      * :cppreference-algorithm:`replace`
-   * * :cpp:func:`hpx::parallel::v1::replace_if`
+   * * :cpp:func:`hpx::replace_if`
      * Replaces all values satisfying specific criteria with another value.
      * ``<hpx/algorithm.hpp>``
      * :cppreference-algorithm:`replace`
```

Comment:
The same change needs to be applied for `replace_copy` and `replace_copy_if`.

Reply:
@hkaiser  I will solve this in the next docs commit ;)